src/Microsoft.DotNet.Watcher.Tools/Internal/MsBuildProjectFinder.cs

```diff
@@ -35,20 +34,20 @@ public static string FindMsBuildProject(string searchBase, string project)
 
                 if (projects.Count > 1)
                 {
-                    throw new GracefulException(Resources.FormatError_MultipleProjectsFound(projectPath));
+                    throw new FileNotFoundException(Resources.FormatError_MultipleProjectsFound(projectPath));
```

Comment:
Seemed like the most suitable type. But we could always create our own type, i.e. `ProjectNotFoundException`